frontend/pages/hosts/details/DeviceUserPage/_styles.scss

```diff
@@ -75,6 +75,14 @@
         }
       }
 
+      &__disk-space-wrapper {
+        display: inline-block;
+
+        &:hover {
+          cursor: help;
```

Comment:
Individual `cursor: help` attributes may be able to be removed based on feedback in that PR.

Reply:
Ahh yeah, I'll work on making the help cursor more global, this is a particular bar graph that I can put a global classname on so it'll work in all 3 instances (manage host page, device user page, host details page)